src/System.Private.CoreLib/shared/System/Threading/WaitHandle.cs

```diff
@@ -305,7 +304,7 @@ private static int WaitMultiple(ReadOnlySpan<WaitHandle> waitHandles, bool waitA
                 {
                     IntPtr[] unsafeWaitHandles = new IntPtr[waitHandles.Length];
                     ObtainSafeWaitHandles(waitHandles, safeWaitHandles, unsafeWaitHandles);
-                    waitResult = context.Wait(unsafeWaitHandles, waitAll, millisecondsTimeout);
+                    waitResult = context!.Wait(unsafeWaitHandles, waitAll, millisecondsTimeout);
```

Comment:
@jaredpar, just FYI, another case where the non-nullness of a local is tracked by a bool, but the compiler isn't able to see that.